packages/compass-aggregations/src/components/pipeline-results-workspace/pipeline-results-header.tsx

```diff
@@ -1,44 +1,69 @@
 import React from 'react';
-import { css, spacing } from '@mongodb-js/compass-components';
+import { css, Overline, spacing } from '@mongodb-js/compass-components';
 import type { ResultsViewType } from './pipeline-results-list';
 import PipelinePagination from './pipeline-pagination';
 import PipelineResultsViewControls from './pipeline-results-view-controls';
 import { connect } from 'react-redux';
 import type { RootState } from '../../modules';
 import { changeViewType } from '../../modules/aggregation';
 import { getStageOperator, isOutputStage } from '../../utils/stage';
+import { PipelineOutputOptionsMenu } from '../pipeline-output-options-menu';
+import type { PipelineOutputOption } from '../pipeline-output-options-menu';
 
 type PipelineResultsHeaderProps = {
   onChangeResultsView: (viewType: ResultsViewType) => void;
   resultsViewType: ResultsViewType;
   isMergeOrOutPipeline: boolean;
+  onChangePipelineOutputOption: (val: PipelineOutputOption) => void;
+  pipelineOutputOption: PipelineOutputOption;
 };
 
-const controlsStyles = css({
+const containerStyles = css({
   display: 'flex',
-  gap: spacing[2],
-  justifyContent: 'flex-end',
+  justifyContent: 'space-between',
```

Comment:
`margin-left: auto` instead of this is exactly something that allows you not to render weird empty divs to preserve the element alignment

Reply:
helped to remove the empty div. thanks!